common/app/navigation/NavLinks.scala

```diff
@@ -110,6 +110,7 @@ object NavLinks {
 
   /* SPORT */
 
+  private val euro2024 = NavLink("Euro 2024", "football/euro-2024", Some("football/euro-2024"))
```

Comment:
Should have been this!

```suggestion
private val euro2024 = NavLink("Euro 2024", "/football/euro-2024", Some("football/euro-2024"))
```